src/Microsoft.AspNetCore.Server.IntegrationTesting/Deployers/ApplicationDeployerFactory.cs

```diff
@@ -35,7 +35,7 @@ public static IApplicationDeployer Create(DeploymentParameters deploymentParamet
                     return new IISExpressDeployer(deploymentParameters, loggerFactory);
                 case ServerType.IIS:
                     throw new NotSupportedException("The IIS deployer is no longer supported");
-                case ServerType.WebListener:
+                case ServerType.HttpSys:
```

Comment:
You'll need to make a PR to music store due to this namechange.

Reply:
And about a dozen other repos. I'll work through those once IIS and ServerTests are happy.